dev/com.ibm.ws.webcontainer/src/com/ibm/wsspi/webcontainer/WebContainerRequestState.java

```diff
@@ -231,40 +235,56 @@ public void setCookieAttributes(String cookieName, String cookieAttributes) {
             logger.logp(Level.FINE, CLASS_NAME, methodName, " cookieName --> " + cookieName + " cookieAttribute --> " + cookieAttributes);
         }
 
-        //future cookieAttributes can be further separated with semicolon delimiter:  attributeName=attributeValue;attributeName2=attributeValue2;singleAttributeNameNoValue
-        //Currently ignore all but SameSite 
+        //Currently ignore all but SameSite or Partitioned
         String[] attribute = cookieAttributes.split("=");
-        if (!attribute[0].equals("SameSite")) {
-                logger.logp(Level.FINE, CLASS_NAME, methodName, " Only SameSite attribute is supported at this time.");
+        if (!(attribute[0].equals("SameSite") || attribute[0].equals("Partitioned"))) {
+                logger.logp(Level.FINE, CLASS_NAME, methodName, " Only SameSite attribute and Partitioned attribute are supported at this time.");
                 return;
         }
+
+        // Should we throw any exceptions? such as: 
+        // if(attribute.length == 1) { // No attributeValue set
+        //     throw IllegalStateException("attribute value not specififed");
+        // }
```

Comment:
This would require a API change since setCookieAttributes doesn't throw any exceptions.